apps/desktop-wallet/src/storage/addresses/addressesActions.ts

```diff
@@ -79,9 +78,8 @@ export const syncAddressesData = createAsyncThunk<
     await dispatch(syncAddressesTokensBalances(addresses))
     return await dispatch(syncAddressesTransactions(addresses)).unwrap()
   } catch (e) {
-    posthog.capture('Error', { message: 'Synching address data' })
```

Comment:
I'll bring it back now that we have throttling but also fixed the loop issue

Reply:
Syncing* yes :D